sklearn/feature_extraction/_hash.py

```diff
@@ -84,8 +85,8 @@ class FeatureHasher(TransformerMixin, BaseEstimator):
     DictVectorizer : vectorizes string-valued features using a hash table.
     sklearn.preprocessing.OneHotEncoder : handles nominal/categorical features.
     """
-
-    def __init__(self, n_features=(2 ** 20), input_type="dict",
+    @_deprecate_positional_args
+    def __init__(self, *, n_features=(2 ** 20), input_type="dict",
```

Comment:
I feel `n_features` could be positional, no strong feeling though.

Reply:
I am +0.25 on having `n_features` positional.